rules/falco_rules.yaml

```diff
@@ -1182,7 +1185,8 @@
     spawned_process
     and container
     and proc.is_exe_upper_layer=true 
-    and not container.image.repository in (known_drop_and_execute_containers)
+    and not known_drop_and_execute_containers
```

Comment:
Supportive. However `and not container.image.repository in (known_drop_and_execute_containers)` shoudl not be changed. Mind reverting that change?